Electron-Basic-Video-Chat/README.md

```diff
@@ -106,22 +108,22 @@ optional parameters:
 * The completion handler
 
 ```
-const publisher = OT.initPublisher('publisher', (error) => {
-  if (error) {
-    return console.log('There was an error initializing the publisher', error);
-  }
-});
+  const publisher = OT.initPublisher('publisher', (error) => {
+    if (error) {
+      return console.log(`There was an error initializing the publisher: ${error}`);
+    }
+  });
 
 ```
 
 Once the Publisher object is initialized, we publish to the session using the `publish()`
 method of the Session object:
 
-session.publish(publisher, (error) => {
-  if (error) {
-    return console.log('There was an error publishing', error);
-  }
-});
+  session.publish(publisher, (error) => {
```

Comment:
This is still not in a code block.

Open the file in the Github UI to see the rendered markdown view.